core/shared/src/main/scala/zio/prelude/Ord.scala

```diff
@@ -157,7 +157,7 @@ object Ord extends Lawful[Ord] {
   /**
    * The set of all laws that instances of `Ord` must satisfy.
    */
-  val laws: Laws[Ord] =
+  lazy val laws: Laws[Ord] =
```

Comment:
Could `lazy val laws` be enough? Do we need to make the individual laws lazy too? The individual laws could be private for all we care, no?
`laws` on the other hand have a dependency on each other in the hierarchy, though.
Just thinking out loud...

Reply:
I think for our purposes here yes but contributors or users could potentially want to test an individual law and could run into the same problem. Maybe erring a bit on the side of caution but I think the cost is minimal (if a `val` versus a `lazy val` is relevant to your performance you shouldn't be using this level of abstraction at all) and there can be really hard bugs to track down.

Reply:
makes sense :+1: